phpstan.neon

```diff
@@ -423,3 +423,14 @@ parameters:
             message: '#"%s" in sprintf\(\) format must be quoted#'
             paths:
                  - packages/monorepo-builder/packages/release/src/Command/ReleaseCommand.php #110
+
+        -
+            message: '#Content of method ".*" is duplicated with method in ".*" class, use unique content instead#'
+            paths:
+                 - packages/php-config-printer/src/DependencyInjection/Extension/PhpConfigPrinterExtension.php #14
+                 - packages/phpstan-rules/src/Rules/PreventDuplicateClassMethodRule.php #73
+                 - packages/skipper/src/DependencyInjection/Extension/SkipperExtension.php #14
+                 - packages/smart-file-system/src/Json/JsonFileSystem.php #33
+                 - packages/smart-file-system/src/Json/JsonFileSystem.php #41
+                 - packages/easy-coding-standard/src/DependencyInjection/Extension/EasyCodingStandardExtension.php #14
+                 - packages/console-package-builder/tests/HttpKernel/ConsolePackageBuilderKernel.php #24
```

Comment:
Seeing these 2 classes, they cannot be extracted from, as it's basic building brick of application.

There should be a list with excluded classes:


```php
/**
 * @var string[]
 */
private const EXCLUDED_TYPES = [
    \Symfony\Component\HttpKernel\Kernel::class,
	\Symfony\Component\DependencyInjection\Extension\Extension::class,
];


// pseudo-code

$class = $this->getClass($scope);
foreach (self::EXCLUDED_TYPES as $excludedType)) {
   if (is_a($class, $excludedType, true)) {
       return [];
   }
}
```

Reply:
I will try.

Reply:
implemented.